src/Concerns/HandlesImports.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Blueprint\Concerns;
+
+use Blueprint\Contracts\Model;
+
+trait HandlesImports
+{
+    protected function addImport(Model $model, $class)
+    {
+        $this->imports[$model->name()][] = $class;
+    }
+
+    protected function buildImports(Model $model)
+    {
+        return collect($this->imports[$model->name()])
```

Comment:
Short closure is breaking 7.3 compatibility. Also `unique` and `sort` could be done before map for a minor optimization.